### DISCUSSION TITLE
Are there any plans to fix the Windows 10 task view issue?

### DISCUSSION THREAD

**Discussion by brx6995** [valinet/ExplorerPatcher#1933] [Q&A]
Are there any plans to fix the Windows 10 task view issue? I have personally tried every workaround and/or solution that has been posited in the various issues & discussion posts and nothing has fixed the issue yet. But my reports and complaints keep getting closed as "not planned" or "duplicates" despite no fix to the issue. I heavily rely on both the Windows 10 taskbar functionality and the task view to perform daily functions. Will this be fixed any time soon?

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio
